sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -65,12 +65,12 @@ export class CryptographyClient {
    *
    * Example usage:
    * ```ts
-   * let client = new CryptographyClient(url, key, credentials);
+   * let client = new CryptographyClient(keyVaultKey, credentials);
    * let result = await client.encrypt("RSA1_5", Buffer.from("My Message"));
    * ```
-   * @param algorithm The algorithm to use
-   * @param plaintext The text to encrypt
-   * @param options Additional options
+   * @param {JsonWebKeyEncryptionAlgorithm} algorithm The algorithm to use.
```

Comment:
Get the latest from master, I believe this type got renamed as part of https://github.com/Azure/azure-sdk-for-js/commit/a20a2e9922a610d817536c2897e1cf2384ba699c#diff-99f43a8279f1fc1dccedeba61ee2469d